Author: Andreas Tille <tille@debian.org>
Date: Mon, 14 May 2012 08:38:03 +0200
Bug-Debian: http://bugs.debian.org/672744
Description: Fixes FTBFS with gcc-4.7 by simply
 following the advise given by gcc

--- old/cufflinks-2.0.2/src/lemon/bits/base_extender.h
+++ new/cufflinks-2.0.2/src/lemon/bits/base_extender.h
@@ -359,10 +359,10 @@
 		}
 		
 		Node source(const UEdge& edge) const {
-			return aNode(edge);
+			return this->aNode(edge);
 		}
 		Node target(const UEdge& edge) const {
-			return bNode(edge);
+			return this->bNode(edge);
 		}
 		
 		void firstInc(UEdge& edge, bool& dir, const Node& node) const {
